### ISSUE TITLE
Some games don't run with mangohud, or the hud doesn't work at all, when launched outside of gamescope

### ISSUE THREAD

**Issue by joker-119** [flightlessmango/MangoHud#1449]
**Describe the bug**
Some games through WINE (specifically the ones I tested were WoW classic and era), freeze with a black screen when launching with mangohud.
Some games (Minecraft) mangohud just doesn't exist for at all, for some reason.
Some games mangohud works perfectly as expected.

All of the above issues only occur when using mangohud OUTSIDE of gamescope. 

For example:
`mangohud wine WowClassic.exe` results in the game freezing on a black screen indefinitely.
But `gamescope -W 2560 -H 1440 --mangoapp -- wine WowClassic.exe` works fine, both the game and mango load up and there's no issues.

Another example:
`mangohud java -jar minecraft.jar ...` results in MC loading up, but with no mangohud present.
But `gamescope -W 2560 -H 1440 --mangoapp -- java -jar minecraft.jar ...` works fine.

I really want to use mangohud, but using it with gamescope is just not a viable option for me due to it's own bugs (like games refusing to grab the cursor when they should, unless you specify --force-grab-cursor, which wrecks any special sensitivity settings you have, as well as makes the cursor behave unexpectedly when alt-tabbing).

**List relevant hardware/software information**
- Fedora 40
- 0.7.1
- 6700 XT

**To Reproduce**
Varies

**Expected behavior**
mangohud should not be reliant on gamescope for consistent operation.

**Comment by joker-119** [comment]
It seems to only be an issue with native wine, proton via steam seems to work alright, but using mangohud on wine causes the graphics pipeline to die. Sound still works but graphics don't and the program gets marked as "not responding".

I've manually updated to v 0.7.2 (didn't realize the fedora repo was outdated) and no change in behavior.

It remains broken when trying to use it on minecraft.


Again, the issues only persist when NOT using gamescope.

**Comment by gianmarcotoso** [comment]
I am encountering this issue as well, and it appears to be completely random. I've encountered it with Elden Ring (Steam), Observer System Redux (GOG) and other games as well, while others (Silent Hill 2 Remake, Banishers) work fine. Always when using Proton (either via Steam or UMU Launcher). One thing I've noticed is that is starts happening after a while: I've played Elden Ring A LOT before this started happening, and two days ago I started Observer without issues while today it won't start at all.

Disabling Mangohud solves the issue. I can't comment on Gamescope because it always crashes instantly on Nvidia.

**Comment by Ramen-LadyHKG** [comment]
Somehow Titanfall 2 doesn't work now, it was working a month ago ( I am not quite sure which version was working).

But I test 0.7.0~.0.7.2, no luck on me.

I'm using `umu-launcher` btw (`umu-launcher version 1.1.3`  `UMU-PROTON`

```
Operating System: Arch Linux 
KDE Plasma Version: 6.2.2
KDE Frameworks Version: 6.7.0
Qt Version: 6.8.0
Kernel Version: 6.11.4-arch1-1-surface (64-bit)
Graphics Platform: Wayland
Processors: 8 × Intel® Core™ i7-8650U CPU @ 1.90GHz
Memory: 15.5 GiB of RAM
Graphics Processor: Mesa Intel® UHD Graphics 620
Dedicated Graphics: Nvidia GTX 1060 Mobile
```

**Comment by Ramen-LadyHKG** [comment]
> Somehow Titanfall 2 doesn't work now, it was working a month ago ( I am not quite sure which version was working).
> 
> But I test 0.7.0~.0.7.2, no luck on me.
> 
> I'm using `umu-launcher` btw (`umu-launcher version 1.1.3` `UMU-PROTON`
> 
> ```
> Operating System: Arch Linux 
> KDE Plasma Version: 6.2.2
> KDE Frameworks Version: 6.7.0
> Qt Version: 6.8.0
> Kernel Version: 6.11.4-arch1-1-surface (64-bit)
> Graphics Platform: Wayland
> Processors: 8 × Intel® Core™ i7-8650U CPU @ 1.90GHz
> Memory: 15.5 GiB of RAM
> Graphics Processor: Mesa Intel® UHD Graphics 620
> Dedicated Graphics: Nvidia GTX 1060 Mobile
> ```

Basically all EA Games doesn't work (eg. Battlefield 4, Sim City 4)
But Team Fortores 2, Garry Mod, Outlast working.

**Comment by nonetrix** [comment]
I made a super simple raylib "game" and that doesn't work either

**Comment by flightlessmango** [comment]
Reopen if still an issue

**Comment by laviphon** [comment]
This is still a problem on Linux under both native Linux exports AND Windows exports ran through WINE

**Comment by NaterEST** [comment]
Running a CachyOS system with an RX 9060 XT, and a Ryzen 7 5800XT.
Games like Dragon's Dogma: Dark Arisen, Clone Drone in the Danger Zone, Mothergunship, Sniper Elite 3 and some others that I've tried, mangohud just doesn't work on for me.
House Flipper 2, R.E.P.O, Climbey, BRUTALISTICK VR, Boneworks, Climbey, Vertigo Remastered, Squirrel with a Gun, Road to Vostok and more just work.
Have tried both the env variable `MANGOHUD=1 %command%`, and just putting `mangohud %command%, neither work, a few games actually didn't even want to launch with the env variable. I don't see any visible errors in steam's terminal window, same ignored messages on both games where mangohud works and also on those where it doesn't. It's quite inconsistent, where it doesn't work, but from the looks, majority of the games that I've ran with VKD3D, mangohud works, but more games don't work that use DXVK instead, don't work (ofc a lot of games still work, just more frequently don't seem to)

**Comment by flightlessmango** [comment]
@NaterEST some of those games are 32bit, make sure you have 32bit mangohud installed

**Comment by NaterEST** [comment]
> [@NaterEST](https://github.com/NaterEST) some of those games are 32bit, make sure you have 32bit mangohud installed

Hmm, I was quite sure I installed the 32 bit libraries for mangohud, I guess I will make sure to check that when I'm back on my pc. Will reply back when and if I've checked if I have lib32-mangohud installed

**Comment by NaterEST** [comment]
yup, apparantly I was indeed missing the lib32 package. Thought I had installed already, but guess I had forgotten.
